.github/workflows/code-coverage.yaml

```diff
@@ -0,0 +1,70 @@
+name: code-coverage
+
+on:
+  push:
+    branch: development
```

Comment:
Is there any reason to not run this on `main` as well?

Reply:
Good call, added!